CMakeLists.txt

```diff
@@ -32,7 +32,7 @@ ExternalProject_Add(
   project_depot_tools
 
   GIT_REPOSITORY    https://chromium.googlesource.com/chromium/tools/depot_tools.git
-  GIT_TAG           4eaefabc5dca618fc07ac550fda35ba7a45a38f8
+  GIT_TAG           495b23b39aaba2ca3b55dd27cadc523f1cb17ee6
```

Comment:
This is very important to resolve issue of goma not found inside depot tool ! introduced this morning